### DISCUSSION TITLE
PHP Fatal error:  Uncaught Symfony\Component\Filesystem\Exception\IOException

### DISCUSSION THREAD

**Discussion by venkateshwarans** [EasyEngine/easyengine#1659] [Support Requests]
I am facing the error as shown in the below image while I am doing any `ee site` command. 

Though I was able to create a simple website with `ee site create example.com`, I am unable to delete it. Facing. the same error as well. 

1. I am using macOS Monterey
2. installed easyengine using `homebrew`

![image](https://user-images.githubusercontent.com/38975782/150622181-2d2eb467-fff9-4a6d-99f2-2c1b43915b7c.png)

**Comment by releez** [comment]
A month old post but Im having the same problem. 
Did you find any solution to this? 
Do you also have the macbook pro with M1 chip? 
I did thought it was some kind of issue with write and read permission with ee and docker but when I fixed everything I could think of it still didn't work. :-(
Maybe I missed something or it's something else that I don’t t understand. 
Could it be a bug in Easyengine?

**Reply by venkateshwarans** [reply]
No. I could not find any solution. I ended up using xampp. Yes. I have a Macbook pro M1 chip.

**Comment by releez** [comment]
I think I found a solution to this problem. 
According to my logs it has problem with establish connection to the database. It search for container "services_global-db_1" but my container is named "services-global-db-1". So it worked for me to rename (docker rename ) the container to "services_global-db_1". After that I could do everything I wanted to do. The error you get and I also have I think is because a file is not created that symfony is trying rename. According to what I understand about symfonys code it isn't checking if the file exists that its trying to rename there for you get a uncaught fatal error. 
I have tried to report a Issue about it. Never done it before.
